app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Api
+  module V1
+    class ReservationsController < ApplicationController
+      before_action :authorize_request
+      before_action :read_reservation, only: %i[show destroy]
+
+      def index
+        reservations = Reservation.where(user_id: params[:user_id])
+        render json: reservations
+      end
+
+      def show
+        # reservation = Reservation.find(params[:id])

```

Comment:
OK, I understood the refactored code and It's good to keep it DRY. Let's delete all commented code.

Reply:
@wandergithub, yes, it was for that. Thank you for your comments !